chrome/global/browser.css

```diff
@@ -95,10 +95,11 @@ the menu bar. */
   /* Remove sidebar border when not in full screen. */
   :not(:root[inFullscreen] #navigator-toolbox[style*="margin-top"]) + #browser #sidebar-splitter {
     border-inline-width: 0 !important;
+    min-width: 6px !important;
```

Comment:
Seems to add extra spacing between the sidebar and browser window. The spacing should be even and consistent throughout the frame.